bandit.js

```diff
@@ -0,0 +1,141 @@
+require('dotenv').config({silent: true})
+
+var Connection = require('./connection')
+
+
+class Bandit {
+  constructor() {
+    // NOTE: Be very careful here. If all references to SLACK_ID are not accounted for
+    // it will result in an infinite loop.
+    this.SLACK_ID = "<@U017GNL24JY>"
```

Comment:
The `start` method on the connection will return the bot's id: https://github.com/patricksmith/good_bot/blob/55e0770f9be88e0ee237f44f474c27ce4128dcc2/bots/guess_bot.js#L23-L27. Can we use that here instead of requiring it to be specified in code?